src/client/mechanisms.rs

```diff
@@ -17,7 +17,7 @@ pub trait Aes256Cbc: CryptoClient {
         wrapping_key: KeyId,
         key: KeyId,
     ) -> ClientResult<'_, reply::WrapKey, Self> {
-        self.wrap_key(Mechanism::Aes256Cbc, wrapping_key, key, &[])
+        self.wrap_key(Mechanism::Aes256Cbc, wrapping_key, key, &[], None)
     }
```

Comment:
Because currently it does not have any effect for `Aes256Cbc`.  I would implement that in a separate PR that then also updates the shortcut functions.